extensions/amp-story/1.0/amp-story-tooltip.js

```diff
@@ -144,9 +149,27 @@ export class AmpStoryTooltip {
       }
     });
 
+    this.checkAndRemoveClickableItemsInFirstPage_();
+
     return this.shadowRoot_;
   }
 
+  /**
+   * Clickable items are not allowed on the first amp-story page. Remove
+   * those items if they are found.
+   * @private
+   */
+  checkAndRemoveClickableItemsInFirstPage_() {
```

Comment:
Is this expected to be in this PR?

If yes, we should think of where this code should live, and make sure it's going to work as you keep enabling interactive elements (it should probably use `TOOLTIP_TRIGGERABLE_SELECTORS`).

Reply:
My point is mostly that I don't think this implementation will work. The way it is done right now, it will try removing elements from the DOM once the tooltip is built, after the user clicks on the element.

Reply:
Yeah that's true.. I guess it could make sense to include it in the grid-layer. I'll leave it for another PR.